package.json

```diff
@@ -60,17 +60,17 @@
     "eslint-plugin-prettier": "^3.1.1",
     "express": "^4.17.1",
     "husky": "^4.2.5",
-    "jest": "^26.6.3",
+    "jest": "^27.3.1",
     "jest-canvas-mock": "^2.2.0",
     "jest-dev-server": "^5.0.3",
     "jest-environment-jsdom": "^26.0.1",
     "jest-junit": "^12.0.0",
     "local-web-server": "^4.2.1",
-    "playwright": "^1.0.2",
+    "playwright": "^1.16.3",
     "prettier": "^2.1.1",
     "shx": "^0.3.2",
     "superagent": "^5.2.2",
-    "ts-jest": "^26.5.3",
+    "ts-jest": "^27.0.7",
```

Comment:
Upgrade here because I am leveraging the `jest/circus-runner` test runner for retries.

Reply:
I REALLY want to make sure our tests aren't flaky in the first place. If they become flaky, I'd rather fix it at the core, rather than get into the habit of rerunning them until they pass. For me, flaky tests are unacceptable.